docs: added CODE_OF_CONDUCT.md (#4143)
expressjs/express: CODE_OF_CONDUCT.md

# Contributor Covenant Code of Conduct

As a member of the Open JS Foundation, Express has adopted the [Contributor Covenant 2.0][homepage].
You can see the full source below [coppied directly from here](https://www.contributor-covenant.org/version/2/0/code_of_conduct.md).

> If an issue arises and you cannot resolve it directly with the parties involved, you can report it to one of the following:

Reviewer: I might use the words from the CPC CODE_OF_CONDUCT.md, and tweak a bit as we want the first option to be to report to the project:

```suggestion
If an issue arises and you cannot resolve it directly with the parties involved, you can report it to the Express project TC through the following email:  XXXX
 
In addition, the OpenJS Foundation maintains a Code of Conduct Panel (CoCP). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled. In order to escalate to the CoCP send an email to coc-escalation@lists.openjsf.org.
```

> - An Express project TC member

Reviewer: ```suggestion
```

> - The OpenJSF team for managing CoC escalations ([see here](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md))

Reviewer: ```suggestion
```
